Dockerfile

```diff
@@ -1,6 +1,8 @@
 # Update the base image in Makefile when updating golang version. This has to
 # be pre-pulled in order to work on GCB.
-FROM golang:1.16.4 as build
+FROM golang:1.17.0 as build
+
+RUN apt-get update && apt-get --no-install-recommends install -y libcap2-bin && apt-get clean && rm -rf /var/lib/apt/lists/* 
```

Comment:
Why we need libcap2-bin? Was there any changes in golang image?

Reply:
yeah, when we use new golang image, can't use setcap command

Reply:
![image](https://user-images.githubusercontent.com/69306452/131523074-2922761b-30c0-48ee-a395-783253ad0b20.png)